### ISSUE TITLE
[Bug]: E2E test failed many times reporting 'status = StatusCode.RESOURCE_EXHAUSTED'

### ISSUE THREAD

**Issue by NicoYuan1986** [milvus-io/milvus#21781]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version: 3849ce5
- Deployment mode(standalone or cluster):standalone
- MQ type(rocksmq, pulsar or kafka):    rocksmq
- SDK version(e.g. pymilvus v2.0.0rc2):2.3.0.dev21
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```

### Current Behavior

E2E test failed many times reporting 'status = StatusCode.RESOURCE_EXHAUSTED, details = grpc: received message larger than max'.
details:
1. Link: https://jenkins.milvus.io:18080/blue/organizations/jenkins/Milvus%20HA%20CI/detail/PR-21766/6/pipeline
2. failed case: test_annoy_index
3. error message:
```
[2023-01-18T02:29:25.078Z] [2023-01-18 02:29:16 - ERROR - pymilvus.decorators]: RPC error: [batch_insert], <MilvusException: (code=1, message=<_MultiThreadedRendezvous of RPC that terminated with:
[2023-01-18T02:29:25.078Z] 	status = StatusCode.RESOURCE_EXHAUSTED
[2023-01-18T02:29:25.078Z] 	details = "grpc: received message larger than max (102533581 vs. 67108864)"
[2023-01-18T02:29:25.079Z] 	debug_error_string = "{"created":"@1674008956.238086092","description":"Error received from peer ipv4:10.255.78.192:19530","file":"src/core/lib/surface/call.cc","file_line":966,"grpc_message":"grpc: received message larger than max (102533581 vs. 67108864)","grpc_status":8}"
[2023-01-18T02:29:25.079Z] >)>, <Time:{'RPC start': '2023-01-18 02:29:13.566427', 'RPC error': '2023-01-18 02:29:16.239061'}> (decorators.py:108)
[2023-01-18T02:29:25.079Z] [2023-01-18 02:29:16 - ERROR - ci_test]: Traceback (most recent call last):
[2023-01-18T02:29:25.079Z]   File "/usr/local/lib/python3.8/site-packages/pymilvus/decorators.py", line 50, in handler
[2023-01-18T02:29:25.079Z]     return func(self, *args, **kwargs)
[2023-01-18T02:29:25.079Z]   File "/usr/local/lib/python3.8/site-packages/pymilvus/client/grpc_handler.py", line 386, in batch_insert
[2023-01-18T02:29:25.079Z]     raise err
[2023-01-18T02:29:25.079Z]   File "/usr/local/lib/python3.8/site-packages/pymilvus/client/grpc_handler.py", line 376, in batch_insert
[2023-01-18T02:29:25.079Z]     response = rf.result()
[2023-01-18T02:29:25.079Z]   File "/usr/local/lib/python3.8/site-packages/grpc/_channel.py", line 744, in result
[2023-01-18T02:29:25.079Z]     raise self
[2023-01-18T02:29:25.079Z] grpc._channel._MultiThreadedRendezvous: <_MultiThreadedRendezvous of RPC that terminated with:
[2023-01-18T02:29:25.079Z] 	status = StatusCode.RESOURCE_EXHAUSTED
[2023-01-18T02:29:25.079Z] 	details = "grpc: received message larger than max (102533581 vs. 67108864)"
[2023-01-18T02:29:25.079Z] 	debug_error_string = "{"created":"@1674008956.238086092","description":"Error received from peer ipv4:10.255.78.192:19530","file":"src/core/lib/surface/call.cc","file_line":966,"grpc_message":"grpc: received message larger than max (102533581 vs. 67108864)","grpc_status":8}"
[2023-01-18T02:29:25.079Z] >
[2023-01-18T02:29:25.079Z] 
[2023-01-18T02:29:25.079Z] The above exception was the direct cause of the following exception:
[2023-01-18T02:29:25.079Z] 
[2023-01-18T02:29:25.079Z] Traceback (most recent call last):
[2023-01-18T02:29:25.079Z]   File "/home/jenkins/agent/workspace/tests/python_client/utils/api_request.py", line 26, in inner_wrapper
[2023-01-18T02:29:25.079Z]     res = func(*args, **_kwargs)
[2023-01-18T02:29:25.079Z]   File "/home/jenkins/agent/workspace/tests/python_client/utils/api_request.py", line 57, in api_request
[2023-01-18T02:29:25.079Z]     return func(*arg, **kwargs)
[2023-01-18T02:29:25.079Z]   File "/usr/local/lib/python3.8/site-packages/pymilvus/orm/collection.py", line 425, in insert
[2023-01-18T02:29:25.079Z]     res = conn.batch_insert(self._name, entities, partition_name,
[2023-01-18T02:29:25.079Z]   File "/usr/local/lib/python3.8/site-packages/pymilvus/decorators.py", line 109, in handler
[2023-01-18T02:29:25.079Z]     raise e
[2023-01-18T02:29:25.079Z]   File "/usr/local/lib/python3.8/site-packages/pymilvus/decorators.py", line 105, in handler
[2023-01-18T02:29:25.079Z]     return func(*args, **kwargs)
[2023-01-18T02:29:25.079Z]   File "/usr/local/lib/python3.8/site-packages/pymilvus/decorators.py", line 136, in handler
[2023-01-18T02:29:25.079Z]     ret = func(self, *args, **kwargs)
[2023-01-18T02:29:25.079Z]   File "/usr/local/lib/python3.8/site-packages/pymilvus/decorators.py", line 56, in handler
[2023-01-18T02:29:25.079Z]     raise MilvusException(message=str(e)) from e
[2023-01-18T02:29:25.079Z] pymilvus.exceptions.MilvusException: <MilvusException: (code=1, message=<_MultiThreadedRendezvous of RPC that terminated with:
[2023-01-18T02:29:25.079Z] 	status = StatusCode.RESOURCE_EXHAUSTED
[2023-01-18T02:29:25.079Z] 	details = "grpc: received message larger than max (102533581 vs. 67108864)"
[2023-01-18T02:29:25.079Z] 	debug_error_string = "{"created":"@1674008956.238086092","description":"Error received from peer ipv4:10.255.78.192:19530","file":"src/core/lib/surface/call.cc","file_line":966,"grpc_message":"grpc: received message larger than max (102533581 vs. 67108864)","grpc_status":8}"
[2023-01-18T02:29:25.079Z] >)>
[2023-01-18T02:29:25.079Z]  (api_request.py:39)
[2023-01-18T02:29:25.079Z] [2023-01-18 02:29:16 - ERROR - ci_test]: (api_response) : <MilvusException: (code=1, message=<_MultiThreadedRendezvous of RPC that terminated with:
[2023-01-18T02:29:25.079Z] 	status = StatusCode.RESOURCE_EXHAUSTED
[2023-01-18T02:29:25.079Z] 	details = "grpc: received message larger than max (102533581 vs. 67108864)"
[2023-01-18T02:29:25.079Z] 	debug_error_string = "{"created":"@1674008956.238086092","description":"Error received from ...... (api_request.py:40)
```

### Expected Behavior

pass

### Milvus Log

[artifacts-milvus-standalone-PR-21766-6-pymilvus-e2e-logs.tar.gz](https://jenkins.milvus.io:18080/job/Milvus%20HA%20CI/job/PR-21766/6/artifact/artifacts-milvus-standalone-PR-21766-6-pymilvus-e2e-logs.tar.gz)

**Comment by xiaofan-luan** [comment]
@NicoYuan1986 
the rpc is too large. Milvus only accept batch less than 64M. but now the size is 102533581

**Comment by NicoYuan1986** [comment]
I presume that the error is associated with my pr (update helm charts).
I'll ask the developers concerned to figure it out.

**Comment by NicoYuan1986** [comment]
Some Settings and test cases are wrong when charts update to '4.0.6'.
I'll find the cases that need to be modified. And the settings will be modified after the holiday.

**Comment by NicoYuan1986** [comment]
Fixed by #21766
